akka-actor/src/main/java/akka/annotation/ApiMayChange.java

```diff
@@ -33,4 +33,11 @@
   ElementType.TYPE,
   ElementType.PACKAGE
 })
-public @interface ApiMayChange {}
+public @interface ApiMayChange {
+
+  /**
+   * Reference to issue discussing the future evolvement of this API
+   */
+  String issue() default "";
```

Comment:
Not a fan of creating an empty string for each if unused but... java.

Reply:
It wont create an empty string, because it is a compile time annotation

Reply:
Forgot about that, thanks.